extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -190,92 +167,105 @@ export class MssqlObjectExplorerNodeProvider extends ProviderBase implements sql
 		}
 	}
 
-	private getSessionForNode(node: TreeNode): Session {
-		let rootNode: DataServicesNode = undefined;
-		while (rootNode === undefined && node !== undefined) {
+	private getSqlClusterSessionForNode(node: TreeNode): SqlClusterSession {
+		let sqlClusterSession: SqlClusterSession = undefined;
```

Comment:
Minor: why did we need to change this logic? Didn't it already work? Did something break?